app/Auth/TotpAuth.php

```diff
@@ -117,13 +117,13 @@ public function getSecret()
      * @param  string $label
      * @return string
      */
-    public function getQrCodeUrl($label)
+    public function getQrCodeUrl($label, $options)
     {
         if (empty($this->secret)) {
             return '';
         }
 
-        return GoogleAuthenticator::getQrCodeUrl('totp', $label, $this->secret);
+        return GoogleAuthenticator::getQrCodeUrl('totp', $label, $this->secret,false, $options);
```

Comment:
According to the signature of the method `GoogleAuthenticator::getQrCodeUrl()` the fourth argument is an integer not a boolean, the default is null.